python/MDSplus/tree.py

```diff
@@ -311,7 +311,9 @@ def getShotDB(expt,path=None,lower=None,upper=None):
         expt = str(expt).lower()
         def getTreePath():
             # split path variable into single paths and replaces the ~t treename
-            expt_paths = _mds.getenv(expt+'_path')
+            expt_paths = _mds.getenv('%s_path' % expt)
```

Comment:
Isn't the % format method deprecated in python3?

Reply:
https://realpython.com/python-string-formatting/

given that teh newstyle is not supported in 2.6 but we still support 2.6 this is a legit was of doing it.

Reply:
We should drop support for 2.6, it's ancient and out of support, and then start using the new way to be more in line with python3